openml_OS/models/api/v1/Api_data.php

```diff
@@ -589,18 +590,34 @@ private function data_edit() {
       }
     }
 
-    $update_result = $this->Dataset->update($data_id, $update_fields);
-    // If result returns error    
-    if( $update_result == false ) {
-      $this->returnError( 1067, $this->version );
-      return;
+    // Add description in the description table as a new version
+    if (isset($update_fields['description']))
+    {
+      $description_record = $this->Dataset_description->getWhereSingle('did =' . $data_id, 'version DESC');
+      $version_new = $description_record->version + 1;
+
+      $desc = array(
+          'did'=>$data_id,
+          'version' => $version_new,
+          'description'=>$update_fields['description'],
+          'uploader' => $dataset->uploader);
+
+      unset($update_fields['description']);
+      $desc_id = $this->Dataset_description->insert($desc);
+      if (!$desc_id) {
+        $this->returnError(1067, $this->version);
+        return;
+      }
     }
-    
-    // update elastic search index.  
-    try {
```

Comment:
i like that this is removed!! but where does it go?

Reply:
Sorry this was a merge mistake. I merger my changes from the docker branch based on master (In master this is removed I think). I should not be removing this. (Will fix it with other changes)